api/server/sdk/volume_migrate.go

```diff
@@ -31,18 +31,47 @@ func (s *VolumeServer) Start(
 ) (*api.SdkCloudMigrateStartResponse, error) {
 
 	if volume := req.GetVolume(); volume != nil {
+		// Check ownership
+		if err := checkAccessFromDriverForVolumeId(ctx, s.driver(), volume.GetVolumeId(), api.Ownership_Read); err != nil {
+			return nil, err
+		}
+
 		//migrate volume
 		return s.volumeMigrate(ctx, req, volume)
 	} else if volumeGroup := req.GetVolumeGroup(); volumeGroup != nil {
+
+		labels := make(map[string]string, 0)
+		labels["group"] = volumeGroup.GetGroupId()
+		if !s.haveOwnership(ctx, labels) {
+			return nil, status.Error(codes.InvalidArgument, "Volume Operation not Permitted.")
```

Comment:
I don't know what happened to my review, but this needs to be `codes.PermissionDenied`